Sunrise/Models/Attribute.swift

```diff
@@ -20,4 +20,27 @@ struct Attribute: Mappable {
         value              <- map["value"]
     }
 
+    // MARK: - Value string representation
+
+    func value(type: AttributeType) -> String? {
+        guard let typeName = type.name else { return nil }
+
+        switch (typeName, value) {
+            case ("boolean", let value as Bool):
+                return value ? NSLocalizedString("Yes", comment: "Yes") : NSLocalizedString("No", comment: "No")
+            case (let typeName, let value as String) where ["text", "enum"].contains(typeName):
+                return value
+            case (let typeName, let value as [String: String]) where ["ltext", "lenum"].contains(typeName):
+                return value.localizedString
+            case ("number", let value as Int):
+                return String(value)
+            case ("number", let value as Double):
+                return String(value)
+            case ("money", let value):
+                return Mapper<Money>().map(value)?.description
+
```

Comment:
Here's an example for the `Set` type.

JSON snippet from the product type:
```json
      "attributes": [{
        "type": {
          "name": "set",
          "elementType": {
            "name": "reference",
            "referenceTypeId": "product"
          }
        },
        "name": "similarProducts",
        ....
```

JSON snippet from a product:
```json
"masterVariant": {
        "prices": [],
        "attributes": [{
          "name": "similarProducts",
          "value": [{
            "typeId": "product",
            "id": "<product-1-id>"
          }, {
            "typeId": "product",
            "id": "<product-2-id>"
          }]
        }]
      }
```